app/controllers/works_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 # Controller for a Work
 class WorksController < ApplicationController # rubocop:disable Metrics/ClassLength
-  before_action :set_work, only: %i[show edit update destroy review]
+  before_action :set_work_and_collection, only: %i[show edit update destroy review]
```

Comment:
I was surprised that `set_work` was setting a collection ivar as well, so I thought this would be a painless tweak.